packages/@okta/vuepress-site/docs/reference/api/devices/index.md

```diff
@@ -0,0 +1,855 @@
+
+#### Permitted OAuth2 scopes 
+`okta.devices.manage`
+
+#### Request Path Parameters
+
+| Parameter   | Type   | Description                                                             |
+| ----------- | ------ | ----------------------------------------------------------------------- |
+| `deviceId`  | String | The `id` of [Device](#device-object) object                              |
+
+#### Request Query Parameters
+
+None
+
+#### Request Body
+
+None
+
+#### Response Body
+
+```http
+HTTP/1.1 204 No Content
+```
+
+#### Usage Example
+
+##### Request
+
+```bash
+curl -v -X DELETE \
+-H "Authorization: SSWS ${api_token}" "https://{yourOktaDomain}/api/v1/devices/{deviceId}"
+```
+
+##### Response
+
+```http
+HTTP/1.1 204 No Content
+Content-Type: application/json
+```
+
+##### Error Response
+
+Passing an invalid `id` returns a `404 Not Found` status code with error code `E0000007`.
+
+## Lifecycle Operations
+
+Device lifecycle is defined as transitions of the [Device Status](#device-status) by the associated operations. The Device object follows a predefined lifecycle transition flow. Device Lifecycle operations are idempotent. These are synchronous calls.
+
+* [Activate Device](#activate-device)       
+* [Deactivate Device](#deactivate-device)  
+* [Suspend Device](#suspend-device)       
+* [Unsuspend Device](#unsuspend-device)   
+
+### Activate Device
+
+<ApiOperation method="post" url="/api/v1/devices/${deviceId}/lifecycle/activate" />
+
+Sets a Device's `status` to `ACTIVE`.
+
+Activated devices can be used to create and delete Device User links.
+
+#### Permitted OAuth2 scopes 
+`okta.devices.manage`
+
+#### Request Path Parameters
+
+| Parameter   | Type   | Description                                                             |
+| ----------- | ------ | ----------------------------------------------------------------------- |
+| `deviceId`  | String | The `id` of [Device](#device-object) object                              |
+
+#### Request Query Parameters
+
+None
+
+#### Request Body
+
+None
+
+#### Response Body
+
+None
+
+#### Usage Example
+
+##### Request
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://{yourOktaDomain}/api/v1/devices/{deviceId}/lifecycle/activate"
+```
+
+##### Response
+
+```http
+HTTP/1.1 204 No Content
+Content-Type: application/json
+```
+
+#### Error Responses
+
+* Passing an invalid `id` returns a `404 Not Found` status code with error code `E0000007`.
+* Passing an `id` that is not in the `CREATED` or `DEACTIVATED` status returns a `400 Bad Request` status code with error code `E0000001`.
+
+### Deactivate Device
+
+<ApiOperation method="post" url="/api/v1/devices/${deviceId}/lifecycle/deactivate" />
+
+Sets a Device's `status` to `DEACTIVATED`. Deactivation will cause a Device to lose all Device User Links. A Device should be in `DEACTIVATED` status before it can be [deleted](#delete-device).
```

Comment:
```suggestion
Sets a Device's `status` to `DEACTIVATED`. Deactivation causes a Device to lose all Device User links. A Device should be in `DEACTIVATED` status before it can be [deleted](#delete-device).
```